src/Flicking.ts

```diff
@@ -1084,6 +1084,10 @@ class Flicking extends Component<FlickingEvents> {
       return Promise.reject(new FlickingError(ERROR.MESSAGE.ANIMATION_ALREADY_PLAYING, ERROR.CODE.ANIMATION_ALREADY_PLAYING));
     }
 
+    if (this._control.holding) {
```

Comment:
Is it correct to do moveTo while dragging?

Reply:
This fixes an issue that causes `moveTo` to throw an error when the input is holding on Flicking.

However, `control.animating` is false at `HOLDING` state, but becomes true in `DRAGGING` state.
When in DRAGGING state, `moveTo` throws an `ANIMATION_ALREADY_PLAYING` error above. 

It has been changed to prevent unhandled error by releasing user input using `release`, Should we block `moveTo` while in the holding state?